apps/explorer/src/app/components/nav/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+import routerConfig from '../../routes/router-config';
+
+export const Nav = () => {
+  return (
+    <nav>
+      {routerConfig.map((r) => (
+        <div key={r.name}>
+          {/* TODO stupid hack */}
+          <Link to={r.path.replace('/*', '')}>{r.name}</Link>
```

Comment:
TODO here? Do all the routes need the /* on the end or should the router config have another property for the nav link path?